packages/compass-import-export/src/import/analyze-csv-fields.ts

```diff
@@ -163,8 +162,6 @@ export function analyzeCSVFields({
       delimiter,
       header: true,
       step: function (results: Papa.ParseStepResult<PapaRowData>, parser) {
-        debug('analyzeCSVFields:step', results);
```

Comment:
I removed a bunch of this debugging because it is all in a tight loop (once per row or even once per csv cell) and that really messes with performance plus it just floods the console. The performance only really effects development, but it is nice to be able to get a sense of how this would perform "in real life". We can just add back any debugging we need temporarily when we need it.